.github/workflows/ci.yml

```
@@ -206,3 +206,25 @@ jobs:
         uses: re-actors/alls-green@release/v1
         with:
           jobs: ${{ toJSON(needs) }}
+
+  docker-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ vars.GHCR_USER }}
+          password: ${{ secrets.GHCR_TOKEN }}
+
+      - name: Build Docker Image
+        run: |
+          docker build -t ghcr.io/kadena.js/graph:latest .
+          cd packages/apps/graph && docker compose build graphql-source -t ghcr.io/kadena/graph:dev -t ghcr.io/kadena/graph:${GITHUB_SHA::7}
```

This one should only release for `:dev` and `:commit_hash`, as this one runs only in PRs.

So the `docker build` with `:latest` is not applicable here, right?

```suggestion
      - name: Build Docker Image
        run: |
          cd packages/apps/graph && docker compose build graphql-source -t ghcr.io/kadena/graph:dev -t ghcr.io/kadena/graph:${GITHUB_SHA::7}
```